koin-projects/koin-core/src/main/kotlin/org/koin/core/KoinApplication.kt

```diff
@@ -40,8 +40,8 @@ class KoinApplication private constructor() {
      * Load definitions from modules
      * @param modules
      */
-    fun modules(modules: Module): KoinApplication {
```

Comment:
I would like to keep the one module version, vararg & list if possible

Reply:
Completed 👍